doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -12,31 +12,34 @@ The ``Scheme`` command is a high-level tool for generating automatically
 (possibly mutual) induction principles for given types and sorts. Its
 syntax follows the schema:
```

Comment:
This sentence is repeated in the description of the command below, so I would simply remove it.